uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/utils/LogChecker.java

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.fuse.ui.bot.tests.utils;
+
+import static org.junit.Assert.assertTrue;
+
+import org.jboss.tools.fuse.reddeer.LogGrapper;
+
+public class LogChecker {
+	
+	public static boolean noFuseError() {
+		return (LogGrapper.getPluginErrors("fuse").size() == 0);
+	}
+	
+	public static void assertFuseError() {
```

Comment:
I would have named it assertNoFuseError. I think that it reflects more what it is doing.